docs/IE_DG/Int8Inference.md

```diff
@@ -1,15 +1,35 @@
 # Low-Precision 8-bit Integer Inference {#openvino_docs_IE_DG_Int8Inference}
 
-## Table of Contents
-1. [Supported devices](#supported-devices)
-2. [Low-Precision 8-bit Integer Inference Workflow](#low-precision-8-bit-integer-inference-workflow)
-3. [Prerequisites](#prerequisites)
-4. [Inference](#inference)
-5. [Results analysis](#results-analysis)
+## Introduction
+The goal of this document is to describe some aspects of low-precision inference using OpenVINO Inference Engine (IE). 
+Currently, many Intel HW devices support 8-bit arithmetic out-of-the-box. It allows boosting inference performance 
+significantly by loading more data into a single processor instruction and executing more arithmetical operations at
+one time. The real performance improvement from 8-bit inference depends on the HW but theoretically it can achieve **4X**
+ compared with FP32 execution and **2X** with FP16 at some cost of accuracy which is negligible in many cases.
 
-## Supported devices
+## Low-Precision 8-bit Integer Inference Workflow
+Low-precision workflow consists of two basic steps:
+- *Model preparation*. This step includes model quantization using the tools from OpenVINO ecosystem such as:
+  - [Post-training Optimization Toolkit](@ref pot_README)
+  - [Neural Network Compression Framework](https://github.com/openvinotoolkit/nncf)
+  Please refer to the [Low Precision Optimization Guide](@ref pot_docs_LowPrecisionOptimizationGuide) 
+for more details about the model optimization workflow. 
+- *Low-precision 8-bit inference*. At this step, the quantized model undergoes a set of transformations (the so-called 
+`Low Precision Transformations`) that convert it 
+to the representations that are used by the OpenVINO™ plugins to map the quantized operations to low-level low-precision 
+inference primitives. When you pass the quantized IR to the OpenVINO™ plugin, the plugin automatically recognizes it as
+a quantized model. An indicator of it is [FakeQuantize](../ops/quantization/FakeQuantize_1.md) operation that provides 
+the quantization parameters and scheme. If you pass a quantized model to the plugin that does not support 8-bit
+inference but supports all operations from the model, the model is inferred in the floating-point precision that this
+plugin supports.
```

Comment:
I think this is confusing/unclear: "the model is inferred in the floating-point precision that this
plugin supports". At least I don't understand what you mean here. If it supports FP32 and FP16, how the model will be inferred? Maybe it's better to just say "the model is inferred in the floating-point precision".